forage-android/src/main/java/com/joinforage/forage/android/network/ForageConstants.kt

```diff
@@ -18,6 +18,11 @@ internal object ForageConstants {
 
     object RequestBody {
         const val CARD_NUMBER_TOKEN = "card_number_token"
+        const val REASON = "reason"
+        const val METADATA = "metadata"
+        const val POS_TERMINAL = "pos_terminal"
+        const val POS_TERMINAL_ID = "pos_terminal_id"
```

Comment:
```suggestion
        const val POS_PROVIDER_TERMINAL_ID = "provider_terminal_id"
```

I think this is supposed to be `provider_terminal_id`